compass/landice/tests/ismip6_run_ais/ismip6_run_ais.cfg

```diff
@@ -0,0 +1,44 @@
+# config options for ismip6 antarctic ice sheet data set
+[ismip6_run_ais]
+
+# Resolution that will be set up.  Should be one of 08, 04 with leading 0.  The paths below must manually be set to be consistent
+mesh_res = 08
+
+# COMPASS load script path
+# This is user-specific.
+compass_load_path = /global/cscratch1/sd/hoffman2/AIS8km/compass/load_dev_compass_1.2.0-alpha.1_cori-haswell_gnu_mpt_albany.sh
+
+# Base path to the processed input ismip6 ocean and smb forcing files. User has to supply.
+forcing_basepath = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/forcing/ais_mesh_8to30km_res
+
+# Path to the initial condition file. User has to supply.
+init_cond_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/relaxation_0TGmelt_10yr_muCap/AIS_8to30km_r01_20220906.smooth3.basinsFineTuned_carvedRonne_CIRWIP_relaxation_0TGmelt_10yr_muCap.nc
+
+# Path to the file for the basal melt parametrization coefficients.
+melt_params_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/basin_and_coeff_gamma0_DeltaT_quadratic_non_local.nc
+
+# Path to the region mask file
+region_mask_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027/AIS_8to30km_r01_20220607.regionMask_ismip6.nc
+
+# Path to the directory containing graph files
+graph_files_path = /global/project/projectdirs/piscees/MALI_projects/ISMIP6-2300/initial_conditions/AIS_8to30km_20221027
+
+# True if running coupled MALI-sea level model simulation
+sea_level_model = True
+
+# NOTE: for the directories related to the sea-level model input/outputs, the slash '/' at the end of the directory name is necessary. 
+# Path to the directory containing globally defined ice thickness field for the sea-level model 
```

Comment:
It would be much better to append the `/` at the end in the code if it is missing than tell the user they have to remember to add it.